printer_mods/Ferdi360/2.4_Bedfan_Backplates/.metadata.yml

```diff
@@ -0,0 +1,27 @@
+---
+# The title of your mod. Choose a short and descriptive name
+title: 2.4 Bedfan Backplates
+# A short description of your mod. Try to describe your mod with 1-2 short sentences
+description: Backplates for your Voron 2.4 that block hot air out of your bedfans from blowing onto your ACM panels to (slightly) improve airflow and temps. Also a nice way to showoff Voron branding and the serial of your printer.
+# Mod version: Set this to 1 when you are submitting a new mod. When you make changes to your mod, you may increment this number
+mod_version: 1
+# You may add your discord username here so people can contact you about your mod
+discord_username: Ferdi Mac
+# The CAD files for your mod. Please provide them as a list.
+# Note that each mod must include at least one CAD file.
+cad:
+- STEP/2.4_Backplates_v1.f360
```

Comment:
just a minor nitpick to add here: an `f3d` file isn't a STEP file, it would probably make more sense to name the folder `CAD`, but that's just my opinion :)

(if you do this, be sure to change the folder itself, and this metadata file to match.